marklogic-data-hub-central/ui/src/components/xml-view/xml-view.test.tsx

```diff
@@ -0,0 +1,20 @@
+import {getByText, render} from "@testing-library/react";
+import XmlView from "../xml-view/xml-view";
+import React from "react";
+
+describe("XML view detail component - RTL", () => {
+    test('Rendering xml document with es namespace in instance document', () => {
+        const xmlString = "<Document><content><es:envelope xmlns:es=\"http://marklogic.com/entity-services\"><es:instance><Person><fname xsi:type=\"xs:string\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xs=\"http://www.w3.org/2001/XMLSchema\">Alexandra</fname></Person></es:instance></es:envelope></content></Document>";
```

Comment:
Let's move the xmlString to UI/src/assets/mock-data folder, may be under 'common.data.ts' or some other appropriate file and then import here so you can use it in multiple places. It makes the tests look cleaner.

Reply:
The data is very specific to this test. Should the xmlString go into the UI/src/assets/mock-data folder

Reply:
@rahulvudutala general guidelines here: https://wiki.marklogic.com/display/ENGINEERING/Writing+Data+Hub+Tests#WritingDataHubTests-RTLTests

export it in a file and import in the test even if its specific to a test. Later you might have to enhance it to reuse in another test.
Lets call it namespacedXMLInstance instance of xmlString.